js/views/appview.js

```diff
@@ -153,19 +153,22 @@ define(function(require) {
 			OC.Notification.showTemporary(message);
 			$('#mail_message').removeClass('icon-loading');
 		},
-		showSetup: function() {
+		showSetup: function(account) {
```

Comment:
👎 this is way too hacky. why should we pass an account object when we show the setup screen? This is independent of any (existing) account.